EXILED/Exiled.Events/Patches/Events/Scp330/InteractingScp330.cs

```diff
@@ -45,9 +39,6 @@ private static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstructi
 
             LocalBuilder ev = generator.DeclareLocal(typeof(InteractingScp330EventArgs));
 
-            // Remove original "No scp can touch" logic.
-            newInstructions.RemoveRange(0, 3);
-
```

Comment:
This was a feature :3 then i made

Reply:
So - readd?

Reply:
`That has been returned, presuming the client won't even send the request if it's not a human. If this behavior should be re-introduced, just add the RemoveRange back.`

So add it back: 3

Reply:
👍